src/index.ts

```diff
@@ -33,5 +42,21 @@ void yargs(hideBin(process.argv))
         });
     }
   )
-
+  .command(
+    "list-inactive",
+    "list stacks that appear inactive and can potentially be deleted. Note this uses simple heuristics and false positives are very likely.",
+    (yargs) => {
+      yargs;
+    },
+    async (argv) => {
+      const logLevel = getLogLevel(argv["log-level"] as string);
+      configureLogging(logLevel);
+      await doForAll(Config.AWS_PROFILES, Config.AWS_REGIONS, runListInactive);
```

Comment:
(Suggesting we migrate overview to use this pattern too for handling the profiles/regions combos.